src/components/Gradebook/GradebookFilters/StudentGroupsFilter/__snapshots__/test.jsx.snap

```diff
@@ -0,0 +1,118 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Should we have a snapshot where Cohorts selector is disabled if there are no cohorts?

Reply:
yeah... was holding back because that should just be coming from a selector-based prop, but it makes sense to go ahead and add the test while that conditional is in there.